src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html

```diff
@@ -0,0 +1,38 @@
+<ct-modal-window modal-window-id="CARDS_PANEL">
+
+    <div class="ct-modal-window-bg" ct-modal-window-close></div>
+    <div class="ct-modal-window-content content">
+
+        <button ng-disabled="cards().knight===0">
```

Comment:
you already have enabled/disabled functionality, but if you are going to implement third step: clickable, in such case leave it as is